### PR TITLE
Add footer nav

### PR DESCRIPTION
**Pull request by steinberggit**
Preview: [FooterNav Storybook](https://transmitter-footer-menu-bsx4br.herokuapp.com/storybook/?path=/story/footermenu--footer-transmitter-menu)

### CODE REVIEW

File: client/src/stories/FooterMenu.stories.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  title: 'FooterMenu',
+  decorators: [withKnobs],
+};
+
+export const FooterTransmitterMenu = () => (
+  <FooterMenu
+    links={[
+      {
+        label: 'Event',
+        Icon: Spacecraft,
```
Comment (florianGierlichs) [inline client/src/stories/FooterMenu.stories.js:17]: You could think about an `alt` tag for the icons maybe.

File: client/src/assets/icons/Icons.js [hunk 2]
```diff
@@ -82,6 +82,154 @@ export const Rocket = ({ active }) => {
...
 
+export const Spacecraft = ({ active }) => {
+  return (
+    <SVG
+      version="1.1"
+      xmlns="http://www.w3.org/2000/svg"
+      width="40.0px"
+      height="40.0px"
+      viewBox="-80 0 710 760"
+      active={active}
```
Comment (florianGierlichs) [inline client/src/assets/icons/Icons.js:93]: Why you give the SVGs the active prop? I cant see that you use that prop for anything or do I miss something?

File: client/src/stories/FooterMenu.stories.js (hunk 1 above)
Comment (steinberggit) [reply client/src/stories/FooterMenu.stories.js:17]: II have implemented the `SVG` as an inline element. `alt` attribute is only available for images. I searched for `SVG` and `alt` and found that for `SVG`, as a replacement for `alt`, you could use the `title` tag, which is nested within the `SVG` tag. I implement it.
[Sitepoint accessible SVG](https://www.sitepoint.com/tips-accessible-svg/)

File: client/src/assets/icons/Icons.js (hunk 2 above)
Comment (steinberggit) [reply client/src/assets/icons/Icons.js:93]: The active prop is in use for the color change of the SVG. When you change the knobs text in storybook from 'Event' to 'Play' or 'Stream', the color of the icons changed

**Pull request merged**